### ISSUE TITLE
Custom loss functions and performance metrics break final model evaluation

### ISSUE THREAD

**Issue by tiantheunissen** [MUST-Deep-Learning/KnowIt#267]
Currently, the use can define a completely custom loss function or performance metric and pass it as an argument.

This means that, after training, if run_model_eval is called to evaluate the model, KnowIt will attempt to find the loss function in the model_args.yaml. It is not saved there.

To consider if this should be fixed and how to fix it.
